tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -57,6 +57,15 @@
 
     EOF
 
+- name: Modify Neutron db name
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    {{ mariadb_copy_shell_vars }}
+
+    sed -i '/^CREATE DATABASE/s/ovs_neutron/neutron/g;/^USE/s/ovs_neutron/neutron/g' ovs_neutron.sql
```

Comment:
you can use replace module from builtin to do that.

One problem with `sed` is that you assume it's installed. While it's part of POSIX, it's better to not make such assumptions. (This is a more theoretical problem.)

Another problem with `sed` is that it will return 0 even when no pattern was replaced. I think it's better to fail then than proceed. Because the missing pattern may indicate some problem with the dump. Better fail now than later?

Reply:
Agree, not having sed installed can be a problem. Will try different approach using some builtin module.

Regarding checking if string substitution was done correctly, since we are doing this bc we need neutron db name to be "neutron" I think it's best to check (on a different task after doing the replace) that the db name is "neutron".

But I agree on adding a check ensuring that the final value is correct.

Reply:
Sorry for not jumping in earlier, i was on PTO last week. I'm not against merging what we have here but my feedback would be:

* If sed is pre-installed on the platforms we're interested in (RHEL, CentOS) then we can use it. If we need to specify some pre-requisite package installs for the Adoption procedure, that's fine as long as those packages come from the OS repos and not some additional ones.

* The test suite should follow the docs as closely as possible. We want to test that what we're telling users to run in the docs actually works. That can mean often using `shell` module instead of "pretty Ansible". (We talked about this only in calls, it's missing in the contrib docs, i reported #131 to fix it).

+2 as i don't want to go churning on the review :). Let's merge the DB import fix and we can refactor later.

Reply:
To clarify:

> If we need to specify some pre-requisite package installs for the Adoption procedure, that's fine as long as those packages come from the OS repos and not some additional ones.

^ i wouldn't go too excessive/exotic with such package requirements but basic tools like sed should be totally fine.

Reply:
Didn't know that test suite should follow docs as closely as possible, since I need to create new patch due to typo in variable, I'll address ansible bits and use sed again to mimic docs :)